File: app/src/main/java/com/doubean/ford/data/SizedPhoto.java
package com.doubean.ford.data;

import com.google.gson.annotations.SerializedName;

public class SizedPhoto {
    public String description;
    public String id;
    public SizedImage images;
    public boolean origin;
    @SerializedName("tag_name")
    public String tag;

}

File: app/src/main/java/com/doubean/ford/model/SizedPhoto.kt
package com.doubean.ford.model

data class SizedPhoto(
    val id: String,
    val description: String,
    val image: SizedImage,
    val origin: Boolean,
    val tag: String,
)

data class SizedImage(
    val isAnimated: Boolean = false,
    val large: ImageItem,
    val normal: ImageItem,
    val raw: ImageItem? = null,
    val small: ImageItem? = null,
    val video: ImageItem? = null,
)

data class ImageItem(
    val height: Int,
    val size: Long,
    val url: String,
    val width: Int,
)